### PR TITLE
Add rizin support

### PR DESCRIPTION
**Pull request by ctlayon**
- Added a flare_emu_rizin.py script for rizin support. (mostly s/radare2/rizin/g)
- Updated tests if given 3 arguments it will run the rizin test.
- Updated flare_emu.py analysis helper by adding optional flag to specify rizin.
- Added instructions to readme for rizin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ It currently provides five different interfaces to serve your emulation needs, a
 ## [Installation](#installation)
 To install `flare-emu` for IDA Pro, simply drop `flare_emu.py`, `flare_emu_ida.py`, and `flare_emu_hooks.py` into your IDA Pro's `python` directory and import it as a module in your IDAPython scripts. 
 
+To install `flare-emu` for Rizin , simply ensure that `flare_emu.py`, `flare_emu_rizin.py`, and `flare_emu_hooks.py` are in Python's search path for importing modules. When using Radare2 as the binary analysis component for `flare-emu`, [rzpipe](https://github.com/rizinorg/rz-pipe) is required.
```
Comment (jtbennett-fe) [inline README.md:22]: should "Radare2" be "Rizin" here? copy/paste error?

File: README.md [hunk 2]
```diff
@@ -58,6 +60,48 @@ The `iterateCallback` function receives the EmuHelper instance, named `eh` here,
...
 
+### Easy String Decryption Scenario with Rizin
```
Comment (jtbennett-fe) [inline README.md:63]: thank you very much for including some use case docs!

**Review by jtbennett-fe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ctlayon) [reply README.md]: Fixed!

**Review by jtbennett-fe — APPROVED** [review]
LGTM, thank you for adding Rizin support to the project!

**Pull request merged**